### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by aliquo93** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#791] [General]
As the title says, I'm asking because it hasn't been updated in a year...

**Comment by im-coder-lg** [comment]
Thanks for asking this question. Even I don't know :( it would be sad to see this app gone and moved to archives.

**Comment by Melvin-Abraham** [comment]
Hi @aliquo93 @im-coder-lg, this project isn't dead. The development has slowed down due to other personal commitments but I don't wish to archive it any time soon. You are more than welcome to contribute to the development of this project... 😃

**PS:** A minor version will be released soon

**Comment by clmcgrath** [comment]
is this project going to be affected by the sunsetting of voice driven apps in june ?
